src/main/scala/com/globo/bigdata/ghdfs/HdfsManager.scala

```diff
@@ -34,6 +35,19 @@ class HdfsManager(hdfs: FileSystem) {
 
   def write(hadoopPath: Path): FSDataOutputStream = create(hadoopPath: Path)
 
+  def write(hadoopPath: Path, is: InputStream): Boolean = {
```

Comment:
About the InputStream argument, can you change the variable name from "is" to another thing like "input"?